app/services/action_queue.rb

```diff
@@ -155,6 +155,15 @@ def transaction_payload
       }
     end
 
+    def fake_card_info
+      {
+        card_num:       "4111111111111111",
```

Comment:
411... is your archetypal fake credit card number, but no problem changing it. What about `GOCARD`, or maybe `DIRECTDEBIT`?